modules/installation-initializing.adoc

```diff
@@ -403,8 +401,7 @@ it to install multiple clusters.
 +
 [IMPORTANT]
 ====
-The `install-config.yaml` file is consumed during the installation process. If
-you want to reuse the file, you must back it up now.
+The `install-config.yaml` file is transformed into the Kubernetes manifests during the installation process, and the original file is deleted. If you want to reuse the file, you must back it up now.
```

Comment:
Interesting. I understand this to be the case, but I am guessing this transformation to the manifests you were able to observe in your tests?

Reply:
I have observed this yes. Essentially when you run the command the manifests appear and the installation file is removed. This is change is a way to more clearly describe what is meant by "consumed" in this note.

Reply:
I added a similar update in related PR 28079, which is discussed [in this comment](https://github.com/openshift/openshift-docs/pull/28079#discussion_r560901152). That PR was SME and QE reviewed.

Reply:
@mffiedler please disregard this request. I am going to revert this line back to the original one and create a separate 4.7.z PR that makes that change outside of this PR.

Reply:
@vikram-redhat I tested a custom IPI installation just now. From that test I now understand that the manifests are only relevant to UPI installation scenarios, specifically the `./openshift-install create manifests...` step. In my IPI testing, no `<install_dir>/manifests/` was created. As such, I will not create a new PR for this line change.

Thank you for highlighting this!